internal/daysteps/daysteps.go

```diff
@@ -13,8 +19,48 @@ const (
 
 func parsePackage(data string) (int, time.Duration, error) {
 	// TODO: реализовать функцию
+	splitData := strings.Split(data, ",")
+	if len(splitData) == 2 {
+
+		steps, err := strconv.Atoi(splitData[0])
+		if steps <= 0 {
+			return 0, 0, fmt.Errorf("количество шагов должно быть больше 0")
+		}
+
+		if err != nil {
+			return 0, 0, fmt.Errorf("ошибка при преобразовании количества шагов: %v", err)
```

Comment:
При оборачивании нужно использовать `%w` (wrap), а просто при выводе в консоль `%v`
```suggestion
			return 0, 0, fmt.Errorf("ошибка при преобразовании количества шагов: %w", err)
```